### PR TITLE
Actionable items

### PR DESCRIPTION
**Pull request by dwalintukan**
Gets the `actionableItemCount` for the badge number in the Nav Bar My Activities badge:

This design does not have the number, but we can put it there.
image

Also refactored the Redux items and put the correct actions, sagas, reducers in the right spot.

### CODE REVIEW

File: src/redux/Topic/saga.js [hunk 1]
```diff
@@ -35,17 +36,17 @@ export function* calculateWinningsRequestHandler() {
         type: actions.CALCULATE_WINNINGS_RETURN,
         value,
       });
-    } catch (error) {
+    } catch (err) {
       yield put({
         type: actions.CALCULATE_WINNINGS_RETURN,
-        value: { error: error.message ? error.message : '' },
+        error: err.message,
```
Comment (liviavinci) [inline src/redux/Topic/saga.js]: err.message or err?

File: src/redux/app/reducer.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const initState = new Map({
   current: preKeys,
   walletAddrs: [],
   walletAddrsIndex: 0,
-  selected_wallet_address: 'wtf',
+  selectedWalletAddress: '',
```
Comment (liviavinci) [inline src/redux/app/reducer.js]: lol good

File: src/redux/Topic/saga.js (hunk 1 above)
Comment (dwalintukan) [reply src/redux/Topic/saga.js]: I am returning just the string currently. If we have any use later to have the entire `error` object, will refactor all the instances later.

I wanted to have a catch-all for all errors in the reducers.

File: src/redux/app/reducer.js (hunk 2 above)
Comment (dwalintukan) [reply src/redux/app/reducer.js]: 🤣

**Pull request merged**